test/models/job_test.rb

```diff
@@ -64,6 +64,21 @@ class JobTest < ActiveSupport::TestCase
     job.calculate_retry_delay.must_equal 7.days
   end
 
+  it 'can have retries scheduled' do
+    # set-up preconditions to retry
+    t1 = job.tasks.create!(task_type: 'audio', label: 'test1')
```

Comment:
Useless assignment to variable - `t1`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.